src/testers.jl

```diff
@@ -102,38 +116,48 @@ function frule_test(f, xẋs::Tuple{Any, Any}...; rtol::Real=1e-9, atol::Real=1e
     dΩ_fd = _make_jvp_call(fdm, (xs...) -> f(deepcopy(xs)...; deepcopy(fkwargs)...), Ω, xs, ẋs, ẋs_is_ignored)
     check_equal(dΩ_ad, dΩ_fd; isapprox_kwargs...)
 
-    # No tangent is passed in to test accumlation, so generate one
-    # See: https://github.com/JuliaDiff/ChainRulesTestUtils.jl/issues/66
-    acc = rand_tangent(Ω)
+    acc = output_tangent isa Auto ? rand_tangent(Ω) : output_tangent
     _check_add!!_behaviour(acc, dΩ_ad; rtol=rtol, atol=atol, kwargs...)
 end
 
 
+
 """
-    rrule_test(f, ȳ, (x, x̄)...; rtol=1e-9, atol=1e-9, fdm=central_fdm(5, 1), fkwargs=NamedTuple(), check_inferred=true, kwargs...)
+    test_rrule(f, inputs...; kwargs...)
 
 # Arguments
 - `f`: Function to which rule should be applied.
-- `ȳ`: adjoint w.r.t. output of `f` (should generally be set randomly).
-  Should be same structure as `f(x)` (so if multiple returns should be a tuple)
-- `x`: input at which to evaluate `f` (should generally be set to an arbitary point in the domain).
-- `x̄`: currently accumulated adjoint (should generally be set randomly).
-
-Non-differentiable arguments, such as indices, should have `x̄` set as `nothing`.
-`fkwargs` are passed to `f` as keyword arguments.
-If `check_inferred=true`, then the inferrability of the `rrule` is checked — if `f` is
-itself inferrable — along with the inferrability of the pullback it returns.
-All remaining keyword arguments are passed to `isapprox`.
+- `inputs` either the primal inputs `x`, or primals and their tangents: `x ⊢ ẋ`
```

Comment:
ẋ should be x̄. 
Maybe it should be explained also here how to produce a vdash